internal/engine/actions/remediate/pull_request/pull_request.go

```diff
@@ -200,13 +248,24 @@ func (r *Remediator) Do(
 	var remErr error
 	switch remAction {
 	case interfaces.ActionOptOn:
-		alreadyExists, err := prWithContentAlreadyExists(ctx, r.ghCli, repo, magicComment)
+		alreadyExists, slug, err := prWithContentAlreadyExists(ctx, r.ghCli, repo, magicComment)
 		if err != nil {
 			return nil, fmt.Errorf("cannot check if PR already exists: %w", err)
 		}
 		if alreadyExists {
 			zerolog.Ctx(ctx).Info().Msg("PR already exists, won't create a new one")
-			return nil, nil
+			b, err := json.Marshal(map[string]any{
+				"status": db.RemediationStatusTypesPending,
+				"status_data": map[string]any{
+					"organization": slug.GetOrganization(),
+					"repository":   slug.GetRepository(),
+					"number":       slug.GetNumber(),
+				},
+			})
+			if err != nil {
```

Comment:
If you don't feel strongly about decorating the error, you can also just `return b, err` or even `return json.Marshal(...)`.

Reply:
I usually wrap them as much as I can to know where things broke, but I'm fine return it directly. Done :+1: